.vscode/settings.json

```diff
@@ -0,0 +1,20 @@
+{
+    "files.associations": {
+        "az_http_client.h": "c",
+        "curl.h": "c",
+        "az_curl_adapter.h": "c",
+        "az_http_request.h": "c",
+        "az_mock_curl.h": "c",
+        "az_write_span_iter.h": "c",
```

Comment:
Ehhh, I don't think we want it in source control, unless it's generic for all `.c`, `.h` files.

Reply:
I think it would be good to update .gitignore then.

Reply:
It looks like https://github.com/Azure/azure-sdk-for-c/pull/12 should have it,as well as many other good changes.